packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -134,71 +124,100 @@ export class RemoteTableService {
     input: RemoteTableInput,
```

Comment:
I'm also confused by the "build" naming. I would use "create" instead. I don't know why but build does not persist to me